instrumentation/instaredis/instaredis.go

```diff
@@ -126,27 +127,32 @@ func (h commandCaptureHook) handleHook(ctx context.Context, cmd redis.Cmder, cmd
 	span.Finish()
 }
 
-func (h commandCaptureHook) BeforeProcess(ctx context.Context, cmd redis.Cmder) (context.Context, error) {
-	return ctx, nil
-}
-
-func (h commandCaptureHook) AfterProcess(ctx context.Context, cmd redis.Cmder) error {
-	h.handleHook(ctx, cmd, []redis.Cmder{})
-	return nil
+type InstanaRedisClient interface {
+	AddHook(hook redis.Hook)
+	Options() *redis.Options
 }
 
-func (h commandCaptureHook) BeforeProcessPipeline(ctx context.Context, cmds []redis.Cmder) (context.Context, error) {
-	return ctx, nil
+func (h commandCaptureHook) DialHook(next redis.DialHook) redis.DialHook {
```

Comment:
Could you be so kind and add comments for all exposed methods? Including any existing public ones? I forgot to add them when the original instrumentation was made

Reply:
I have added comments for the methods. But here the `commandCaptureHook` is an unexported type and we are not actually exposing the methods to the user.